CHANGELOG.md

```diff
@@ -403,24 +405,35 @@ Additionally, thanks to [Alexander Turek][@derrabus] for consulting on the repo
     - Thanks to [Jaroslav Hanslík][@kukulich] for the patch
 
 ### Fixed
-- Fixed bug #3502 : A match statement within an array produces Squiz.Arrays.ArrayDeclaration.NoKeySpecified
-- Fixed bug #3503 : Squiz.Commenting.FunctionComment.ThrowsNoFullStop false positive when one line @throw
-- Fixed bug #3505 : The nullsafe operator is not counted in Generic.Metrics.CyclomaticComplexity
+- Fixed bug [#3502] : A match statement within an array produces Squiz.Arrays.ArrayDeclaration.NoKeySpecified
+- Fixed bug [#3503] : Squiz.Commenting.FunctionComment.ThrowsNoFullStop false positive when one line @throw
+- Fixed bug [#3505] : The nullsafe operator is not counted in Generic.Metrics.CyclomaticComplexity
     - Thanks to [Mark Baker][@MarkBaker] for the patch
-- Fixed bug #3526 : PSR12.Properties.ConstantVisibility false positive when using public final const syntax
+- Fixed bug [#3526] : PSR12.Properties.ConstantVisibility false positive when using public final const syntax
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #3530 : Line indented incorrectly false positive when using match-expression inside switch case
-- Fixed bug #3534 : Name of typed enum tokenized as T_GOTO_LABEL
+- Fixed bug [#3530] : Line indented incorrectly false positive when using match-expression inside switch case
+- Fixed bug [#3534] : Name of typed enum tokenized as T_GOTO_LABEL
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #3546 : Tokenizer/PHP: bug fix - parent/static keywords in class instantiations
+- Fixed bug [#3546] : Tokenizer/PHP: bug fix - parent/static keywords in class instantiations
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #3550 : False positive from PSR2.ControlStructures.SwitchDeclaration.TerminatingComment when using trailing   comment
+- Fixed bug [#3550] : False positive from PSR2.ControlStructures.SwitchDeclaration.TerminatingComment when using trailing   comment
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #3575: Squiz.Scope.MethodScope misses visibility keyword on previous line
+- Fixed bug [#3575]: Squiz.Scope.MethodScope misses visibility keyword on previous line
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #3604: Tokenizer/PHP: bug fix for double quoted strings using ${
+- Fixed bug [#3604]: Tokenizer/PHP: bug fix for double quoted strings using ${
     - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
 
+[#3502]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3502
+[#3503]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3503
+[#3505]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3505
+[#3526]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3526
+[#3530]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3530
+[#3534]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3534
+[#3546]: https://github.com/squizlabs/PHP_CodeSniffer/issues/3546
```

Comment:
These are PRs, not issues.

Reply:
Sorry about that. I missed it when testing the links, as GitHub redirects to the correct URL. It is fixed now. I opted to amend the original commit as this was a small documentation fix.

Reply:
Understood and yes, GH redirects, but we should not rely on that.